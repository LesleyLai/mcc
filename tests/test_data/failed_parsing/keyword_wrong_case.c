int main(void) {
    RETURN 0;
}