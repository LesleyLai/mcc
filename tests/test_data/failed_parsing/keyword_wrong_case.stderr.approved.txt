{{filename}}:2:5: Error: use of undeclared identifier 'RETURN'
2 |     RETURN 0;
  |     ^~~~~~

{{filename}}:2:12: Error: expect ';'
2 |     RETURN 0;
  |            ^

